crates/bevy_ui/src/lib.rs

```diff
@@ -135,6 +136,14 @@ impl Plugin for UiPlugin {
                     .ambiguous_with(bevy_text::update_text2d_layout)
                     .ambiguous_with(widget::text_system),
             )
+            .add_system_to_stage(
+                CoreStage::PostUpdate,
+                widget::update_slider_value.label(widget::UpdateSliderValue),
+            )
+            .add_system_to_stage(
+                CoreStage::PostUpdate,
+                widget::update_slider_handle.after(widget::UpdateSliderValue),
+            )
```

Comment:
I feel like we should have a WidgetPlugin where this kind of thing is handled. Otherwise this will get way too big.

Reply:
I think there should also be a SliderWidgetPlugin to keep this as self contained as possible